x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_list_page/index.test.tsx

```diff
@@ -195,6 +195,8 @@ describe('agent_list_page', () => {
           totalInactive: 0,
         },
       });
+      jest.useFakeTimers({ legacyFakeTimers: true });
```

Comment:
This `describe` is trying to use fake timers but this was only applied in the previous `describe` causing warning in this block.